modules/developers-guide/pages/cql.adoc

```diff
@@ -0,0 +1,157 @@
+= Using the CQL API
+
+== Prerequisites
+
+To use the CQL API you need:
+
+include::quickstart:page$quick_start.adoc[tag=prereqsList]
+* cqlsh (standalone, or as part of a separate Cassandra installation)
+
+include::quickstart:page$quick_start.adoc[tag=getDockerImage]
+
+include::quickstart:page$quick_start.adoc[tag=startDocker]
+
+=== Installing standalone cqlsh using Docker
+
+[source,bash]
+----
+docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' stargate
+docker pull nuvo/docker-cqlsh
+----
+
+// Use `pip` to install the standalone cqlsh tool.
+//
+// NOTE: cqlsh requires Python 2.7, and won't work with Python 3.
+//
+// [source,bash]
+// ----
+// pip install cqlsh
+// ----
+
+== Using the CQL API using cqlsh
+
+Run `cqlsh` from the Docker image.
+
+[source,bash]
+----
+docker run --rm -it nuvo/docker-cqlsh cqlsh localhost --cqlversion=3.4.5
```

Comment:
Should we be consistent with username/password here too or just remove from both?

Reply:
Yeah, probably remove from both, since by default auth is off.